pom.xml

```diff
@@ -123,6 +124,21 @@
         <version>${project.version}</version>
         <classifier>forge-addon</classifier>
       </dependency>
+      <dependency>
+        <groupId>io.openshift</groupId>
```

Comment:
The idea was not to add booster-catalog-service as a dependency in root, but in the dependencyManagement section and as a dependency in web and the launcher-addon without the version.

The other modules shouldn't depend on this library